server/src/model/init.ts

```diff
@@ -42,6 +40,17 @@ const addInitialVoteSchema = async () => {
   return schema.save();
 };
 
+const addInitialVoteConfigurationSchema = async () => {
+  const schema = new Parse.Schema("VoteConfiguration");
+  schema.addPointer("board", "Board", {required: true});
+  schema.addNumber("votingIteration", {required: true, defaultValue: 0});
+  schema.addNumber("voteLimit", {required: true, defaultValue: 0});
```

Comment:
Should the default value really be 0? Or something like 10?

Reply:
Yes we can set this to 10.